cmd/fyne/internal/commands/build.go

```diff
@@ -110,7 +120,7 @@ func (b *builder) build() error {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%s\n", string(out))
 	}
-	return err
+	return []string{b.target}, err
```

Comment:
Why is `b.build` returning `b.target`? This state is available to whoever called the build command, and as it is a method that executes change it should probably not return like a getter.